lib/ts/recipe/thirdpartypasswordless/index.ts

```diff
@@ -249,17 +249,21 @@ export default class Wrapper {
         });
     }
 
-    static async sendEmail(input: TypeThirdPartyPasswordlessEmailDeliveryInput & { userContext?: any }) {
```

Comment:
how come thirdPartyManuallyCreateOrUpdateUser is compiling without tenantId as an input?